### ISSUE TITLE
Possible regression through AIX support

### ISSUE THREAD

**Issue by ffrank** [etcd-io/bbolt#195]
We're including bbolt into purpleidea/mgmt. Builds have been failing for some 2 weeks now.

```
# github.com/coreos/bbolt
../../coreos/bbolt/bolt_unix_aix.go:13:6: flock redeclared in this block
	previous declaration at ../../coreos/bbolt/bolt_unix.go:13:59
../../coreos/bbolt/bolt_unix_aix.go:46:6: funlock redeclared in this block
	previous declaration at ../../coreos/bbolt/bolt_unix.go:45:22
../../coreos/bbolt/bolt_unix_aix.go:56:6: mmap redeclared in this block
	previous declaration at ../../coreos/bbolt/bolt_unix.go:50:27
../../coreos/bbolt/bolt_unix_aix.go:76:6: munmap redeclared in this block
	previous declaration at ../../coreos/bbolt/bolt_unix.go:72:21
```

Is this an actual issue, possibly introduced via 7f8bb47fcaf8eaa817f8aad11c9c28b58215b66f?

**Comment by ffrank** [comment]
Paging @amdprophet for feedback maybe?

**Comment by ryupold** [comment]
I have the same problem when using 
`go get -u go.etcd.io/bbolt`
is there a workaround to install it?

**Comment by xiang90** [comment]
ping @amdprophet

**Comment by Kvaz1r** [comment]
Same issue but for Windows. Removing bolt_unix_aix.go helped but it's not really convenient.

**Comment by leobeosab** [comment]
I wrote a fix for this #198

**Comment by xiang90** [comment]
i believe this is fixed by #198. can you give it a try /cc @ffrank ?

**Comment by cenkalti** [comment]
Closing as this issue is very old and not much activity on it. Feel free to reply/re-open.
